private/buf/bufgen/generator.go

```diff
@@ -342,73 +319,6 @@ type remotePluginExecutionResult struct {
 	Index                 int
 }
 
-func (g *generator) executeRemotePlugins(
-	ctx context.Context,
-	container app.EnvStdioContainer,
-	image bufimage.Image,
-	remote string,
-	pluginConfigs []*remotePluginExecArgs,
-	includeImports bool,
-	includeWellKnownTypes bool,
-) ([]*remotePluginExecutionResult, error) {
-	pluginReferences := make([]*registryv1alpha1.PluginReference, len(pluginConfigs))
-	for i, pluginConfig := range pluginConfigs {
-		pluginReference, err := getPluginReference(pluginConfig.PluginConfig)
-		if err != nil {
-			return nil, err
-		}
-		pluginReferences[i] = pluginReference
-	}
-	generateService := connectclient.Make(g.clientConfig, remote, registryv1alpha1connect.NewGenerateServiceClient)
-	response, err := generateService.GeneratePlugins(
```

Comment:
The `GeneratePlugins` endpoint will be removed and has been replaced by `GenerateCode`